changelogs/fragments/54550-Change-windows-ansible-hostname-setup.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "setup.ps1 - Change $env:COMPUTERNAME to [System.Net.Dns]::GetHostName(), to support non NETBIOS compliant hostnames  (https://github.com/ansible/ansible/issues/54550)"
```

Comment:
Needs to be updated, it might be better to leave our implementation changes and just say what was fixed, i.e. non NETBIOS comliant hostnames.